### PR TITLE
90_SIGNALduino_un.pm - PerlCritic / commandref

### PR DESCRIPTION
**Pull request by HomeAutoUser**
* **Please check if the PR fulfills these requirements**

- [ ] Tests for the changes have been added / modified (needed for for bug fixes / features)
- [x] commandref has been added / updated (needed for bug fixes / features)
- [x] CHANGED has been updated (needed for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
  - docs update
  - feature (PerlCritic)

* **What is the current behavior?** (You can also link to an open issue here)
  - any some many PerlCritic warnings

* **What is the new behavior (if this is a feature change)?**
  - some Perl Critic warnings less
  - remove prototypes on subroutine (PerlCritic)
  - replaced " with ' in strings without a variable (PerlCritic)
  - replaced hard tabs (PerlCritic)
  - replaced return undef with return and added return (PerlCritic)
  - revised commandref

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: FHEM/90_SIGNALduino_un.pm [hunk 1]
```diff
@@ -16,296 +16,267 @@ use POSIX;
...
 
-
   $hash->{Match}     = '^[u]\d+(?:.\d)?#.*';
-  $hash->{DefFn}     = "SIGNALduino_un_Define";
-  $hash->{UndefFn}   = "SIGNALduino_un_Undef";
-  $hash->{AttrFn}    = "SIGNALduino_un_Attr";
-  $hash->{SetFn}     = "SIGNALduino_un_Set";
-  $hash->{ParseFn}   = "SIGNALduino_un_Parse";
-  $hash->{AttrList}  = "IODev do_not_notify:0,1 stateFormat showtime:0,1 ignore:0,1 ".$readingFnAttributes;
+  $hash->{DefFn}     = 'SIGNALduino_un_Define';
```
Comment (sidey79) [inline FHEM/90_SIGNALduino_un.pm:26]: Lass uns hier doch gleich coderefs hinterlegen. Ist die sauberste Methode

Comment (HomeAutoUser) [reply FHEM/90_SIGNALduino_un.pm:26]: Ich stehe hier gerade etwas auf dem Schlauch. *sorry*
Was genau meinst du?

Comment (sidey79) [reply FHEM/90_SIGNALduino_un.pm:26]: Gleiches vorgehen wie hier und ff:

https://github.com/RFD-FHEM/RFFHEM/blob/f7ca7bf284cb6bca22ff81f4d9d1b573bdf2b592/FHEM/00_SIGNALduino.pm#L286

Comment (HomeAutoUser) [reply FHEM/90_SIGNALduino_un.pm]: umgesetzt in https://github.com/RFD-FHEM/RFFHEM/pull/914/commits/0aae01a80a4c347f20e2215d49b7b454a399ac8b

**Review by sidey79 — APPROVED** [review]

**Pull request merged**